scipy/sparse/linalg/interface.py

```diff
@@ -338,6 +340,56 @@ def matmat(self, X):
 
         return Y
 
+    def rmatmat(self, X):
+        """Adjoint matrix-matrix multiplication.
+
+        Performs the operation y = A^H * x where A is an MxN linear
+        operator and x is a column vector or 1-d array, or 2-d array.
+        The default implementation defers to the adjoint.
+
+        Parameters
+        ----------
+        x : {matrix, ndarray}
+            A matrix or 2D array.
+
+        Returns
+        -------
+        y : {matrix, ndarray}
+            A matrix or 2D array depending on the type of the input.
+
+        Notes
+        -----
+        This rmatmat wraps the user-specified rmatmat routine or overridden
```

Comment:
I know this is just repeating the boilerplate from the other `Notes` sections, but we should probably rephrase it so that it doesn't reference a private method.

Reply:
removed